### ISSUE TITLE
Support for B2 Application Keys

### ISSUE THREAD

**Issue by ChristofferKarlsson** [gilbertchen/duplicacy#484]
Backblaze recently made B2 Application Keys available, a feature that lets you create application specific keys, and which also can restrict access to specific buckets. Using the B2 account for more than one thing, it would be very nice to use they more restricted keys. I tried using a generated API key (restricted to one only one bucket), but it doesn't seem to work with the current version - 2.1.1:

```
chrille@volgograd $ duplicacy init -e data-backup b2://***
Enter Backblaze Account ID: app-key-id
Enter Backblaze Application Key: app-key
URL request 'https://api002.backblazeb2.com/b2api/v1/b2_list_buckets' returned status code 400
URL request 'https://api002.backblazeb2.com/b2api/v1/b2_list_buckets' returned status code 400
...
```

* [Official Documentation](https://www.backblaze.com/b2/docs/application_keys.html?_ga=2.44038181.1038729054.1534753353-646273881.1534753353)
* [Announcment in Blog](https://www.backblaze.com/blog/b2-application-keys/)

**Comment by TheBestPessimist** [comment]
Is this a duplicate of: https://github.com/gilbertchen/duplicacy/issues/485 ?

**Comment by ChristofferKarlsson** [comment]
@TheBestPessimist Yes, it is about the same feature. Missed that it was already a PR on it.

**Comment by TheBestPessimist** [comment]
Well actually i am wrong: #485 is the duplicate here :D.

**Comment by bekriebel** [comment]
Oops, I missed that someone had opened this as an issue before creating mine. I closed that duplicate issue.

PR #475 is a fix for this and has been open for some time now. I'm hoping it can get merged so I can start using official versions again.

**Comment by TheBestPessimist** [comment]
This has been implemented via: https://github.com/gilbertchen/duplicacy/commit/674d35e5cae603b641a048609ff48335a36ebb36.

**Comment by adrianmihalko** [comment]
Guys, how can I use application key id?

```
@ubuntu:~$ duplicacy init -e ubuntu-backup b2://my-b2-bucket
Enter Backblaze Account ID:312939939393
Enter Backblaze Application Key: llxlxlxlxlxlxlxlxl
Failed to load the Backblaze B2 storage at b2://mihalko-ubuntu-backup: Authorization failure
```

I enter my newly created Application Key, but I get Authorization failure. What I am missing here?
